.github/workflows/release-apps.yml

```
@@ -0,0 +1,71 @@
+name: Release Apps
+
+on:
+  release:
+    types: [ published ]
+  push:
+    branches: [ main ]
+  pull_request_target:
```

```suggestion
  pull_request:
```

I think it should be `pull_request`, since `_target` is a privileged environment.

---

changed to use `pull_request`